build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.7.4-0
```

Comment:
I guess we can modify this tag to include the k8s version in it. maybe v1.7.5-k8s1.5-0?

I should double-check that there aren't any other changes we should cherrypick if we do that.